docs/guides/dashboard-sso-okta-setup.md

```diff
@@ -1,3 +1,17 @@
+---
+id: dashboard-sso-okta-setup
+title: Configuring Okta Single Sign-On (SSO)
+hide_title: false
+hide_table_of_contents: false
+sidebar_label: Configuring Okta Single Sign-On (SSO)
+pagination_label: Configuring Okta Single Sign-On (SSO)
+keywords:
+  - docs
+  - docusaurus
+image: https://i.imgur.com/mErPwqL.png
+slug: /dashboard-sso-okta-setup
+---
+
```

Comment:
@avb-is-me ... Why do we have this meta here?
This is breaking the build. (because docusaurus cannot find the page anymore under /docs/guides/dashboard-sso-okta-setup).

Reply:
@sudobinbash I'll edit it.  Main reason is the bottom:

![Screenshot 2022-12-16 at 5 44 24 PM](https://user-images.githubusercontent.com/104213687/208208937-374dde03-455f-4369-bcac-4389a736d824.png)

The next step in guides just keeps looping back instead of letting the user proceed.